tests/viscoelastic_plastic_brick_extension.prm

```diff
@@ -35,6 +35,31 @@ subsection Initial composition model
   end
 end
 
+# Use viscoelastic plastic material model
+subsection Material model
```

Comment:
When you later update the base (included prm), we should probably remove the material model section here.

Reply:
That is the plan 👍. The `viscoelastic_plastic` test suite will keep the `Material model` section as the corresponding benchmarks will soon use `visco_plastic`, but the intention is to completely remove `viscoelastic_plastic` (source code, tests) soon thereafter.

Reply:
Thanks for handling all of the reviews and merges today!